src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search big-search"
+      data-qa="big"
+      action="/search.php"
+      method="get">
+      <span class="icon__search"></span>
+      <label>
+        <input
+          for="search"
+          class="input__search"
+          data-qa="keypress"
+          placeholder ="Try &quot;Los Angeles&quot">
+        </input>
+      </label>
+    </form>
+
+    <form
+     class="search small-search"
+     data-qa="small"
+     action="/search.php"
+     method="get">
```

Comment:
Fix everywhere. Check 5-th point in [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
```suggestion
    <form
     class="search small-search"
     data-qa="small"
     action="/search.php"
     method="get"
   >
```